Gemfile

```diff
@@ -3,9 +3,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.6.5'
-
-gem 'dotenv-rails', groups: [:development, :test]
+ruby '2.6.6'
```

Comment:
I would argue that since Ruby is specified at the Dockerfile level and we only support running the app inside a container there is no need for this line to exist and that it is not dry

Reply:
d'oh, fair point